index.html

```diff
@@ -0,0 +1,136 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Foursquare Annual conference</title>
+  <link rel="stylesheet" href="styles.css">
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+<link href="https://fonts.googleapis.com/css2?family=Lato:wght@100;400;900&family=Roboto:ital,wght@0,100;0,300;0,400;0,500;0,700;1,300&display=swap" rel="stylesheet">
+<script defer src="https://use.fontawesome.com/releases/v5.0.6/js/all.js"></script>
+<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.2.1/css/all.min.css" integrity="sha512-MV7K8+y+gLIBoVD59lQIYicR65iaqukzvf/nwasF0nqhPay5w/9lJmVM2hMDcnK1OnMGCdVK+iQrJ7lzPJQd1w==" crossorigin="anonymous" referrerpolicy="no-referrer" />
+<script defer src="script.js"></script>
+</head>
+<body>
+  <header>
+    <div class="container-fluid deskto-top-nav">
+      <div class="social-media-div">
+        <ul class="social-media">
+          <li class="">
+            <a href="" class="social-media-item"><i class="fa-brands fa-facebook-f"></i></a>
+          </li>
+          <li class=""><a href="#" class="social-media-item"><i class="fa-brands fa-twitter"></i></a>
+          </li>
+          <li class="">
+            <a href="#" class="social-media-item">English</a>
+          </li>
+          <li class="">
+            <a href="#" class="social-media-item">Mypage</a>
+          </li>
+          <li class="">
+            <a href="#" class="social-media-item">Logout</a>
+          </li>
+        </ul>
+      </div>
+    </div>
+    <nav class="navbar">
```

Comment:
- [ ] Kindly ensure that the background of your navbar on mobile is transparent or the same color/pattern as the top section background which is lavenderblush